Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8893,13 +8897,16 @@ def inner(client, args):
                     else poll_result.continue_to_poll()
                 if not should_poll:
                     return poll_result.response
+
+                readable_output = poll_message if not args.get('hide_polling_output') else None
                 poll_args = poll_result.args_for_next_run or args
-                readable_output = poll_message if not args.get('polled_once') else None
-                poll_args['polled_once'] = True
-                return CommandResults(readable_output=readable_output,
-                                      scheduled_command=ScheduledCommand(command=name, next_run_in_seconds=interval,
+                poll_args['hide_polling_output'] = True
```

Comment:
Why default `True`? I think it'll be better to propagate the `hide_polling_output` that was provided in the args.

Reply:
hide_polling_putput should be false for the first iteration and then true. So the first iteration we will get "Fetching Results:" and from the second iteration and on we get None. This sets it in the polling_args for the second iteration